src/libraries/System.Linq/src/System/Linq/Index.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+
+namespace System.Linq
+{
+    public static partial class Enumerable
+    {
+        /// <summary>Return index and the associated item.</summary>
```

Comment:
This isn't an ideal description of what the method does: the method doesn't return an index and an associated item.